### DISCUSSION TITLE
Tags for Versions

### DISCUSSION THREAD

**Discussion by Meerkap** [coreybutler/nvm-windows#869] [Ideas & Feature Requests] [answered]
Would be interesting to add tags for each version of node installed in order to know what version you used for.
An exmple of this could be:

~ $nvm tags
   - IonicApp_SomeName
   - ReactEcommerce_SomeName
   - TodoList
   - GenericTagForTesting
   - AngularAppForClientX
   - DiscordBot_SomeName

~ $nvm list

```
    18.2.0
          - TodoList
          - ReactEcommerce_SomeName
    18.0.0
    17.0.0
          - GenericTagForTesting
          - DiscordBot_SomeName
    16.14.2
  * 12.20.2 (Currently using 64-bit executable)
          - AngularAppForClientX
          - IonicApp_SomeName
```

**Comment by coreybutler** [comment, marked as answer]
Custom aliases (tags) are on the roadmap for [Runtime](https://github.com/coreybutler/nvm-windows/discussions/565#:~:text=The%20current%20plan%20is%20to%20replace%20NVM4W%20with%20a%20successor%2C%20code%2Dnamed%20%22rt%22%2C%20which%20emphasizes%20%22runtime%20management%22.).
